app/controllers/api/v0/events_controller.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+require 'uri'
+require 'byebug'
```

Comment:
remember to remove before merging

Reply:
Ah, yup. BTW, I don't see lots of other `require` s in other parts of the code - and in fact w/o this, the URI class/module/whatever is available. What's general Rails practice for this sort of thing?

Reply:
Discussed in scrum, fixed in code.